collections/nemo_nlp/nemo_nlp/modules/classifiers.py

```diff
@@ -122,13 +122,15 @@ def create_ports():
     def __init__(self,
                  hidden_size,
                  num_classes,
+                 name=None,
```

Comment:
what's name for?

Reply:
I'm using TokenClassifier twice: once for punctuation, another one for capitalization task. I was getting errors since both classifiers have the same name, somewhere in the nf. Let me know if you think there is a better way to fix it.